docs/getting_started/features/flexible_services.md

```diff
@@ -5,7 +5,281 @@ Flexible services, also called demand-responsibe services, are services that do
 
 Continuous Stops is used when riders can be picked up and/or dropped off between scheduled stops all along the route or on specific sections of it. This is defined in `routes.txt`, indicating that the rider can be picked up of dropped off the transit vehicle at any point along the vehicle’s travel path, on every trip of the route.
 
+**Pre-requirement**: 
+
+- [Base features](/getting_started/features/base)
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
 |[stop_times.txt](/schedule/reference/#stop_timestxt)|`continuous_pickup`, `continuous_drop_off` |
 |[routes.txt](/schedule/reference/#routestxt)|`continuous_pickup`, `continuous_drop_off` |
+
+??? note "Sample Data"
+
+    <p style="font-size:16px">
+    The following sample shows two ways of representing continuous stop, the first one using routes.txt to specify a vehicle that can stop for pick-up and/or drop off at any point along the total length of the route. Secondly, using stop_times.txt to define a specific segment between certain stops where the rider can request to be picked-up and/or dropped off.
+
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#routestxt"><b>routes.txt</b></a> <br>
+        </p>
+
+        | route_id | route_short_name | route_type | continuous_pickup | continuous_drop_off |
+        |----------|------------------|------------|-------------------|---------------------|
+        | RA       |               17 |          3 |                 0 |                   0 |
+
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#stop_timestxt"><b>stop_times.txt</b></a> <br>
+        </p>
+
+        | trip_id | arrival_time | departure_time | stop_id | stop_sequence | continuous_pickup | continuous_drop_off |
+        |---------|--------------|----------------|---------|---------------|-------------------|---------------------|
+        | AWE1    |      6:10:00 |        6:10:00 | TAS001  |             1 |                   |                     |
+        | AWE1    |      6:14:00 |        6:14:00 | TAS002  |             2 |                   |                     |
+        | AWE1    |      6:20:00 |        6:20:00 | TAS003  |             3 |                 0 |                   0 |
+        | AWE1    |      6:23:00 |        6:23:00 | TAS004  |             4 |                 0 |                   0 |
+        | AWE1    |      6:25:00 |        6:25:00 | TAS005  |             5 |                   |                     |
+
+##  Booking rules
+
+Booking rules can be used to enable users to reserve a trip within a demand-responsive. These rules outline the necessary prerequisites for successful bookings and provide contact information where users can make trip reservations. This should be used in conjunction with the On-demand services feature.
```

Comment:
Are we missing something after "demand-responsive"?
Suggestion to replace "with the On-demand services feature" with "with one of the On-demand services features described below" since there are several.

Reply:
I renamed the features in the latest commit following your comments and replacing On-Demand for DRT.